accounts-journey/presentation/ui/AccountsListViewController.swift

```diff
@@ -0,0 +1,199 @@
+//
+//  AccountsListViewController.swift
+//  GoldenSampleApp
+//
+//  Created by Backbase R&D B.V. on 12/10/2023.
+//
+
+import UIKit
+import Combine
+import Resolver
+import BackbaseDesignSystem
+import SnapKit
+
+final class AccountsListViewController: UIViewController {
+    
+    private var viewModel: AccountsListViewModel?
```

Comment:
Why is the viewModel nullable. I don't know the best practices of MVVM in iOS. I look at Batches journey and they did like this: https://github.com/backbase-rnd/bus-business-batches-journey-ios/blob/master/Sources/BusinessBatchesJourney/Components/Confirmation/ConfirmationViewModel.swift
https://github.com/backbase-rnd/bus-business-batches-journey-ios/blob/master/Sources/BusinessBatchesJourney/Components/Confirmation/ConfirmationViewController.swift


It the same approach from this article: https://medium.com/dbs-tech-blog/how-to-successfully-implement-model-view-viewmodel-mvvm-architecture-pattern-in-ios-app-8ef97657f180

Reply:
The approach used here is following what's being used in most journey, the ViewController `bind` method does the assignment / instantiation. 

But this is not cast on stone. I can extract the ViewModel methods into an interface / protocol and make this change